services/webhooks2tasks/src/handlers/githubPush.js

```diff
@@ -19,6 +19,15 @@ async function githubPush(webhook: WebhookRequestData, project: Project) {
     const branchName = body.ref.toLowerCase().replace('refs/heads/','')
     const sha = body.after
 
+    const skip_deploy = body.commits.map( commit => Array.isArray(commit.message.match(/skip deploy|deploy skip/i)))
```

Comment:
let's add a short comment here what this actually does :)

Reply:
The revised, simplified version should be more obvious.